src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -436,6 +437,32 @@ void OrbitMainWindow::SetupTargetLabel() {
                    });
 }
 
+void OrbitMainWindow::SetupStatusBarLogButton() {
+  // The Qt Designer doesn't seem to support adding children to a StatusBar.
+  auto* capture_log_widget = new QWidget(statusBar());
```

Comment:
Nit: You could add a `// NOLINT` or `// NOLINTNEXTLINE` comment to silence clang-tidy. I assume it will complain about the raw `new`.

Reply:
Oh thanks, I like that, added a couple in this method.